recipes/sobjectizer/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(sonjectizer REQUIRED CONFIG)
```

Comment:
Typo. Can also probably replace this and below with add_subdirectory(../test_package).